Add PHP logs correlation documentation (#18610)
DataDog/documentation: content/en/tracing/trace_collection/library_config/php.md

**Default**: `0`<br>
Automatically flush the tracer when all the spans are closed; set to `1` in conjunction with `DD_TRACE_GENERATE_ROOT_SPAN=0` to trace [long-running processes][14].

> `DD_TRACE_APPEND_TRACE_IDS_TO_LOGS`

Reviewer: Need to add `DD_LOGS_INJECTION` and maybe remove `DD_TRACE_APPEND_TRACE_IDS_TO_LOGS` if we decide to

Author: Removed `DD_TRACE_APPEND_TRACE_IDS_TO_LOGS` and its related doc (in its usage as well)

: **INI**: `datadog.trace.append_trace_ids_to_logs`<br>
**Default**: `0`<br>

> Append the logs correlation identifiers to logged messages. Use with caution as the message is modified. See [logs correlation documentation][16] for more information on its usage. Added in version `0.88.0`.

Reviewer 2: ```suggestion
Append the logs correlation identifiers to logged messages. Use with caution because this modifies the message. See [logs correlation documentation][16] for more information on its usage. Added in version `0.88.0`.
```
